src/Rector/Contrib/PHPUnit/SpecificMethodInternalTypeRector.php

```diff
@@ -0,0 +1,80 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Rector\Contrib\PHPUnit;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Scalar\String_;
+use Rector\NodeAnalyzer\MethodCallAnalyzer;
+use Rector\Rector\AbstractRector;
+
+/**
+ * Before:
+ * - $this->assertTrue(is_string($anything));
+ *
+ * After:
+ * - $this->assertInternalType('string', $anything);
+ */
+final class SpecificMethodInternalTypeRector extends AbstractRector
+{
+    /**
+     * @var MethodCallAnalyzer
+     */
+    private $methodCallAnalyzer;
+
+    /**
+     * @var string|null
+     */
+    private $activeFuncCallName;
+
+    public function __construct(MethodCallAnalyzer $methodCallAnalyzer)
+    {
+        $this->methodCallAnalyzer = $methodCallAnalyzer;
+    }
+
+    public function isCandidate(Node $node): bool
+    {
+        $this->activeFuncCallName = null;
+
+        if (! $this->methodCallAnalyzer->isTypesAndMethods(
+            $node,
+            ['PHPUnit\Framework\TestCase', 'PHPUnit_Framework_TestCase'],
+            ['assertTrue']
+        )) {
+            return false;
+        }
+
+        /** @var MethodCall $methodCallNode */
+        $methodCallNode = $node;
+
+        $firstArgumentValue = $methodCallNode->args[0]->value;
+        if (! $firstArgumentValue instanceof FuncCall) {
+            return false;
+        }
+
+        return strpos($firstArgumentValue->name->toString(), 'is_') === 0;
```

Comment:
Here it would be nice to have variable with name, instead of:

```php
$firstArgumentValue->name->toString()
```

rather


```php
$methodName
```

---

As for `strpos (=|!)== 0/true/false`, which is compicated to read, understand and easy to confuse,
there is `Nette\Utils\Strings::startsWith()` method, which is clear just from the name. Please use it.

Reply:
This `nette/utils` package simplifies a lot, definitely gonna use it.